src/content/developers/docs/data-structures/ssz/index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Simple Serialize
+description: Explanation of Ethereum's SSZ format.
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+Simple serialize (SSZ) is the serialization method used on the Beacon Chain. It replaces the RLP serialization used on the execution layer everywhere across the consensus layer except the peer discovery protocol. SSZ is specifically designed to be deterministic and compatible with Merkleization.
```

Comment:
Is there a concise way to describe SSZ that we could add to this? We're currently on describing where it is used and what it replaces.